### PR TITLE
correction d'une faute

### PR DESCRIPTION
**Pull request by hervehobbes**

### CODE REVIEW

File: fr/development/rest.rst [hunk 1]
```diff
@@ -173,7 +173,7 @@ utilisant :php:meth:`RequestHandler::addInputType()`
 RESTful Routing
 ===============
 
-Le Router de CakePHP facilite la connection des routes pour les ressources
+Le Router de CakePHP facilite la connexin des routes pour les ressources
```
Comment (antograssiot) [inline fr/development/rest.rst]: you replace a typo by another one here. it should be `connexion`

**Comment by hervehobbes** [comment]
You are right, sorry!

**Comment by saeideng** [comment]
new PR https://github.com/cakephp/docs/pull/5761

**Comment by saeideng** [comment]
you can add more commits by pushing to the your  branch on `hervehobbes/docs`
and no needed to open new PR

**Pull request closed without merging**